src/cli_callback.c

```diff
@@ -289,7 +230,7 @@ static int upload_file(storj_env_t *env, char *bucket_id, const char *file_path,
                                                           handle,
                                                           progress_cb,
                                                           upload_file_complete);
-
+    STORJ_RETURN_IF_LAST_ERROR(1);
```

Comment:
what is this?

Reply:
This is one of a few macros that check the global variable `STORJ_LAST_ERROR`, which is the error string pointer that we pass into all libuplink calls.

This one returns the argument to the macro if the last error was non-empty.

See storjtests.h.